components/LocaleSelector.vue

```diff
@@ -9,4 +9,16 @@
             </option>
         </select>
     </div>
-</template>
\ No newline at end of file
+</template>
```

Comment:
The data was undefined in this file... we have a follow up issue to eventually wire to Pinia

Reply:
Actually these components are just using empty script boilerplate code. In vue3 it's actually a little different. I have some code coming that's going to change most of these